test/swe2d/test_atmospheric_pressure.py

```diff
@@ -0,0 +1,72 @@
+# Atmospheric pressure test case
+# Constant pressure gradient applied at surface
+# Result compared to analytic solution
+
+from thetis import *
+import pytest
+import numpy as np
+
+
+@pytest.mark.parametrize("nx,dt,max_rel_err", [(10, 1200, 2e-6)])
+def test_pressure_forcing(nx, dt, max_rel_err):
+    lx = 21000
+    ly = 5000
+
+    mesh2d = RectangleMesh(nx, 1, lx, ly)
+
+    # Simulation time
+    t_end = 4*24*3600.
+
+    # bathymetry
+    P1 = FunctionSpace(mesh2d, "DG", 1)
+    bathymetry = Function(P1, name='bathymetry')
+    x = SpatialCoordinate(mesh2d)
+    bathymetry.interpolate(Constant(5.0))
+
+    # bottom drag
+    mu_manning = Constant(2.0)
+
+    # atmospheric pressure
+    atmospheric_pressure = Function(P1, name='atmospheric_pressure')
+    atmospheric_pressure.interpolate(x[0])

```

Comment:
For better test coverage the pressure gradient should not only be in the x direction.